app/scheduleSummaryReport/templates/ScheduleSummaryReportCtrl.html

```diff
@@ -77,8 +77,8 @@
 								<!-- Instructors -->
 								<div ng-show="$index==0">
 									<div class="units-details">
-										<span ng-if="sectionGroup.units > 0">
-											Units: {{ sectionGroup.units }}
+										<span ng-if="sectionGroup.unitsLow > 0">
+											Units: {{ sectionGroup.unitsHigh ? sectionGroup.unitsVariable || (sectionGroup.unitsLow + " - " + sectionGroup.unitsHigh) : sectionGroup.unitsLow }}
```

Comment:
We mess up this logic a lot, so I would instead add a transient helper on the entity in the backend so you just see a displayUnits string property which will be the correct thing to show